docroot/modules/custom/va_gov_backend/src/EventSubscriber/EntityEventSubscriber.php

```diff
@@ -550,6 +551,19 @@ public function toggleFieldOfficeHours(array &$form, FormStateInterface $form_st
     }
   }
 
+  /**
+   * Remove delete button from edit forms.
+   */
+  public function removeDeleteButton(array &$form) {
+    // Users with any role, other than administrator role,
+    // shouldn't be able to delete anything, but should be
+    // able to do everything else with content in VA.gov CMS.
```

Comment:
The last part of this statement has nothing to do with your code and is declaring things your code is not doing.  
```suggestion
    // Users with any role, other than administrator role,
    // shouldn't be able to delete anything.
```